packages/components/addon/components/hds/text/code.hbs

```diff
@@ -3,4 +3,12 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-<Hds::Text @group="code" @size={{this.size}} @weight={{this.weight}} @tag={{@tag}} ...attributes>{{yield}}</Hds::Text>
\ No newline at end of file
+<Hds::Text
+  @group="code"
+  @size={{this.size}}
+  @weight={{this.weight}}
+  @align={{@align}}
+  @color={{@color}}
+  @tag={{@tag}}
```

Comment:
[dust ✨] Should the @tag be added to the `It implements the following APIs:` section of the PR description?

Reply:
@LINDGRENBA the `@tag/@size/@weight` arguments were described in the first PR: https://github.com/hashicorp/design-system/pull/1457